tf_encrypted/protocol/pond.py

```diff
@@ -70,6 +71,9 @@ def __init__(
             if tensorflow_supports_int64():
                 tensor_factory = int64factory
             else:
+                logging.warning(
+                    "Falling back to using int100 tensors due to lack of int64 support. "
+                    "Performance may be improved by installing the latest version of TensorFlow.")
```

Comment:
hmmm kind of misleading, maybe we should link to our documentation on how to install our tensorflow custom builds? tf 1.13 should have int64 support but its not out quite yet, hopefully soon!

Reply:
"by installing a version of TensorFlow supporting this (custom build or 1.13+)."?

Reply:
Yeah sounds good!